paths/graphql/outline.md

```diff
@@ -0,0 +1,24 @@
+---
+
+---
+
+Top-level numbers individual pages?
+
+1. Introduction
+  - Motivation, present the problem that needs solving
+  - API as the solution (context: dynamic, consumable information)
+    - Examples of API usages (format: service, how API is used, link)
+1. First use (jump right in, no later than pg 2)
+  - instructions for navigating to GraphiQL on web UI
+  - interact with interface, show how it works, live data
+  - tell me why, can explain differences between rest and graphql
```

Comment:
perhaps we can also include a section on why you should use API to gather this information instead of just hard coding it every once and a while. For example, why hard link to the repos you are contributing to, when we can use an API to update the list for us kind of thing.